Scruffy.Commands/SlashCommands/LogSlashCommandModule.cs

```diff
@@ -88,12 +88,12 @@ await CommandHandler.PostChart(Context, type, count)
     }
 
     /// <summary>
-    /// Exporting urls of all logs since the given date
+    /// Exporting urls of all logs of the given date
     /// </summary>
-    /// <param name="since">Since day</param>
+    /// <param name="dayString">Since day</param>
     /// <returns>A <see cref="Task"/> representing the asynchronous operation</returns>
     [SlashCommand("export", "Export of log urls")]
-    public Task Export([Summary("since-day", "Logs since the given day. (dd.MM.yyyy)")] string since) => CommandHandler.ExportLogs(Context, since);
+    public Task Export([Summary("day", "Logs of the given day. (dd.MM.yyyy)")] string dayString) => CommandHandler.ExportLogs(Context, dayString);
```

Comment:
If you want to retrict the date range we should add an optional until-day parameter which defaults to +1 day. The function to export all logs since a given day is still required.